packages/frontend/app/(dashboard)/course/[cid]/queue/[qid]/components/QueueInfoColumn.tsx

```diff
@@ -185,22 +208,16 @@ const QueueInfoColumn: React.FC<QueueInfoColumnProps> = ({
               />
             ) : null}
           </Row>
+          {queue?.notes && (
+            <div className="flex max-h-[200px] w-full items-center overflow-y-auto px-2 text-xl text-[#5f6b79] sm:hidden">
+              <Linkify>
+                <div className="min-w-0 whitespace-pre-wrap break-words text-sm italic md:text-base">
+                  {`Notes: ${queue?.notes}`}
+                </div>
+              </Linkify>
```

Comment:
The new notes location kinda felt weird at first but I think it makes a little more sense this way.

Only comment here is if I have no notes, it still says Notes:
![image](https://github.com/user-attachments/assets/302f3168-a33c-4b02-b357-de6bf0e447ad)

Reply:
Yea, I need a stricter check for not just whether its null, but whether it is falsy since and empty string is also falsy in JS unless I'm remembering that wrong. Thanks for pointing this out.